weblate/addons/tests.py

```diff
@@ -95,6 +95,13 @@ def test_create_project_addon(self) -> None:
         self.assertEqual(self.component.project.addon_set.count(), 1)
         self.assertEqual("Test add-on: Test", str(addon.instance))
 
+    def test_create_site_wide_addon(self) -> None:
+        addon = TestAddon.create(acting_user=self.component.acting_user)
```

Comment:
`self.component.acting_user` is probably None at this point, better use self.user instead.